keyboards/gmmk/pro/config.h

```diff
@@ -39,27 +41,33 @@
 #define BOOTMAGIC_LITE_COLUMN 3
 
 #define TAP_CODE_DELAY 10
-#define ENCODERS_PAD_A { C15 }
-#define ENCODERS_PAD_B { C14 }
+#define ENCODERS_PAD_A \
+    { C15 }
+#define ENCODERS_PAD_B \
+    { C14 }
 
 /* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
 #define LOCKING_SUPPORT_ENABLE
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
+#define SW_LINES_ENABLE_DRIVER_1 SW_LINES_ENABLED_1_TO_12
+#define SW_LINES_ENABLE_DRIVER_2 SW_LINES_ENABLED_1_TO_12
```

Comment:
This isn't even correct isn't it?
SW12 is completely disconnected on driver 2.

Not that it makes a difference for functionality, but if you're going to implement disabling of the SW lines this should probably be correct.

Reply:
You are completely right, it should be `#define SW_LINES_ENABLE_DRIVER_2 SW_LINES_ENABLED_1_TO_11`